endpoint/data/server_action.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="server_action_registry_sync" model="ir.actions.server">
+        <field name="name">Sync registry</field>
+        <field name="type">ir.actions.server</field>
+        <field name="model_id" ref="endpoint.model_endpoint_endpoint" />
+        <field name="binding_model_id" ref="endpoint.model_endpoint_endpoint" />
+        <field name="binding_type">action</field>
+        <field name="state">code</field>
+        <field name="code">
+records.filtered(lambda x: not x.registry_sync).write({"registry_sync": True})
```

Comment:
We should remove the `filtered` call. There's a similar fix for this done in shopfloor_base recently.
Please, do it in a separated commit that we can backport ;)